critiquebrainz/frontend/external/musicbrainz_db/__init__.py

```diff
@@ -1,26 +1 @@
-from typing import Optional
-from contextlib import contextmanager
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker, scoped_session, Session
-from sqlalchemy.pool import NullPool
-
-engine = None
-Session: Optional[Session] = None  # noqa: F811
 DEFAULT_CACHE_EXPIRATION = 12 * 60 * 60  # seconds (12 hours)
-
-
-def init_db_engine(connect_str):
-    global engine, Session
-    engine = create_engine(connect_str, poolclass=NullPool)
-    Session = scoped_session(
-        sessionmaker(bind=engine)
-    )
-
-
-@contextmanager
-def mb_session():
```

Comment:
Let's not remove the complete `musicbrainz_db` module. This is why the tests are failing as there are imports to get the `mb_session`. 

The code in `musicbrainz_db` is supposed to create db sessions and use brainzutils module functions to fetch data.